x-pack/plugins/fleet/public/layouts/without_header.tsx

```diff
@@ -12,12 +12,8 @@ import { EuiPage, EuiPageBody, EuiSpacer } from '@elastic/eui';
 export const Wrapper = styled.div`
   background-color: ${(props) => props.theme.eui.euiColorEmptyShade};
 
-  // HACK: Kibana introduces a div element around the app component that results in us
-  // being unable to stretch this Wrapper to full height via flex: 1. This calc sets
-  // the min height to the viewport size minus the height of the two global Kibana headers.
-  min-height: calc(
-    100vh - ${(props) => parseFloat(props.theme.eui.euiHeaderHeightCompensation) * 2}px
-  );
+  // Set the min height to the viewport size minus the height of any global Kibana headers
+  min-height: calc(100vh - var(--euiFixedHeadersOffset));
```

Comment:
This breaks the header spacing if `euiFixedHeadersOffset` is not available. Also I've not been able to figure this out yet. It doesn't work with a fallback of `96px`. You might want to restore the `min-height` value that uses eui props for now. @juliaElastic ☝🏼

Reply:
Thanks for checking, I'll restore it. At some point the old styled-components props will be deprecated/going away, but right now they're frozen during the Emotion conversion and aren't at risk for removal for another 3+ months.

Reply:
Just to close the loop on this comment and the above comments - Ash was unchecking the definition in devtools, which isn't something that will happen in production. I've marked the above code suggestions as resolved and fixed all `var()` usages like this one to have `0` fallbacks in b530f22 🎉